flightmate/src/main/java/com/flightmate/dao/DBUtil.java

```diff
@@ -20,8 +20,15 @@
 
 public class DBUtil {
 	private static final String USERNAME = "root";
-	private static final String PASSWORD = "";
-	private static String M_CONN_STRING = "jdbc:mysql://localhost/";
+	private static final String PASSWORD = "12345678";
```

Comment:
It's best if we all use a consistent password. Everyone else left this blank. I suggest you adjust your "root" user to do the same.